CHANGELOG.md

```diff
@@ -28,12 +28,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   We now wait for 2 seconds unconditionally after the specified message has been found.
 - Support for the `KEEP_CONTAINERS` env variable.
   The functionality of `KEEP_CONTAINERS=true` is superseded by `TESTCONTAINERS=keep`.
-  
+
 ### Fixed
 
-- Removing a docker container did not error if failed. This was fixed by asserting the daemon's response instead of 
-  just the status code: If a docker container was removed correctly using `rm -f -v <ID>` <ID> is printed on stdout. 
+- Removing a docker container did not error if failed. This was fixed by asserting the daemon's response instead of
+  just the status code: If a docker container was removed correctly using `rm -f -v <ID>` <ID> is printed on stdout.
   <ID> can either be the container name or its ID which is used within testcontainer-rs.
+- Fixed clippy warnings of camel case names containing a capitalized acronym.
```

Comment:
Would have been good to mention the actual breaking changes here, i.e. the renaming of `ElasticMq` etc.

Not a blocker for merging though, will adapt in a future PR!

Thanks!